zebra-state/Cargo.toml

```diff
@@ -16,6 +16,8 @@ categories = ["asynchronous", "caching", "cryptography::cryptocurrencies"]
 
 [features]
 
+default = []
```

Comment:
Is this unneccessary?

Reply:
Might be some left-over from previous implementation when feature flags were used.